FESS-GUI/windowsnames.h

```diff
@@ -0,0 +1,7 @@
+#ifndef WINDOWSNAMES
+#define WINDOWSNAMES
+
+#define MAINWINDOW_TITLE "FlyCAM - Flywheel Control and Monitoring"
```

Comment:
It depends defines are typeless and don't use any memory until they are used, converted and processed by the compiler. They also don't need typecasting. If we defined all our text in string they would need to be cast to QString for use in QT. In both instances the program would need to be recompiled if the text in those files was changed. However if this text was stored in a localization file using variables would be a better option.